src/main/kotlin/model/RacingCars.kt

```diff
@@ -1,17 +1,18 @@
 package model
 
-data class RacingCars(private val racingCars: List<RacingCar>) {
+data class RacingCars(val racingCars: List<RacingCar>) {
```

Comment:
If you can use this object effectively, you won't need the `GameHistories` object.